trlx/model/nn/ilql_models.py

```diff
@@ -265,8 +264,8 @@ def generate(
         temperature=1,
         top_k=20,
         logit_mask=None,
-        pad_token_id=50256,
-        eos_token_id=50256,
+        pad_token_id=None,
```

Comment:
Why the change

Reply:
`50256` is the default `eos_token_id` for `gpt2` based tokenizers - this just ensures there is no implicit default for the other non-gpt2 models.